rbi/stdlib/date.rbi

```diff
@@ -340,7 +340,7 @@ class Date
   # DateTime.jd(0,12) + DateTime.new(2001,2,3).ajd
   #                           #=> #<DateTime: 2001-02-03T00:00:00+00:00 ...>
   # ```
-  sig {params(arg0: T.untyped).returns(T.self_type)}
+  sig {params(arg0: T.any(Integer, Float, Rational)).returns(T.self_type)}
```

Comment:
Unfortunately this makes things in Rails a bit harder:

```
Time.zone.now.to_date + 5.days
```

`5.days` is an `ActiveSupport::Duration`, not any of the `Integer`, `Float`, or `Rational`, so constructions like the above no longer type-check.

Any ideas on how to fix this? Changing the type signature to allow `ActiveSupport::Duration` is probably not a good idea.